language/src/main.rs

```diff
@@ -773,6 +840,21 @@ fn main() -> Result<(), usize> {
         None => None,
     };
 
+    let vc = match args.iter().position(|a| a == "--vc-init") {
```

Comment:
Can you make an example somewhere (preferably with CI integration) that uses this and describe how to use it in the book? We can do this in a follow up if you prefer.